lib/facter/ec2_tag_facts.rb

```diff
@@ -105,6 +124,22 @@ def debug_msg(txt)
 
           # append to the hash for structured fact later
           result[name] = tag['value']
+          if name =~ /env/
+		if tag['value'] == "prod"
+			subdomain = "#{region}.#{tag['value']}"
+		else
+			subdomain = "#{tag['value']}"
+		end
+		result["subdomain"] = subdomain
```

Comment:
I don't really like use of the term "subdomain", it's too vague. I thought we were going to keep `env` and `region` and then concatenate them. I don't mind having some concatenation, but I think it should be called region-env, not subdomain.

When the changes are made, please update https://github.com/graduway/psick/pull/87 and then I'll review that as well. Talk to me if anything isn't clear.

Reply:
changed to  result["region_env"]